optimize/backend/src/main/java/io/camunda/optimize/dto/zeebe/process/ZeebeProcessInstanceDataDto.java

```diff
@@ -91,22 +97,22 @@ public long getParentElementInstanceKey() {
 
   @Override
   public BpmnEventType getBpmnEventType() {
-    throw new UnsupportedOperationException("Operation not supported");
+    return bpmnEventType;
```

Comment:
❓ This method was not part of the single jar work like the other 3 changed methods are. Does this one need to be changed? It will always be returning `BpmnEventType.UNSPECIFIED` as the setter is never used

Reply:
I got the list of methods that needed to be implemented (i.e., not to throw exception) by looking at the logs. This one was one of the error cause.

No setter is being used, i.e., even the other fields I just added are not being set by anything optimize-related. I am returning the variable instead of directly the value just for consistency. No strong reason.

I got inspired by a similar commit done in tasklist:
https://github.com/camunda/camunda/commit/c0651f26fefc5c8b26ba13b6677ab5d6ea8036ef#diff-eb77b015387bc1565eb155b69e8d7763a64c4637094f34dc4e1be287d3c29c0bR34

All I am doing is not throw exception, but all of these things are not used by Optimize (that's why initially we implemented them as exceptions, as per Josh's suggestion)

Reply:
Thanks for the explanation! This PR is good to go then! 🎉